### DISCUSSION TITLE
Summarize channel

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#50] [Practical Ideas]
sometimes I can't follow some channels, having a tool to summarize latest happening in channel (or a team!) would be great.

e.g.: I've been out of town for a couple of days, when I get back to mattermost I see a channel I'm interested having a few hundred messages. I type `/summarize 2 days` on a channel and I get a system post or a dm from a bot with links to posts that got most reacted or threads that had a lot of comments, probably a top five of each would be great.

**Comment by esethna** [comment]
Yes a digest of what you missed would be great. See this related one: https://github.com/mattermost/community/discussions/38#discussioncomment-907229

**Comment by Willyfrog** [comment]
> Yes a digest of what you missed would be great. See this related one: https://github.com/mattermost/community/discussions/38#discussioncomment-907229

Yes! I like that home screen idea, it could probably be both, a home screen and a command
